packages/ledit/ledit.2.04/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+authors: "Daniel de Rauglaudre"
+homepage:"http://pauillac.inria.fr/~ddr/ledit/"
+maintainer: "Pierre Boutillier <pierre.boutillier@laposte.net>"
+
+build: make
+depends: ["camlp5"]
+install: [
```

Comment:
There seem to be a problem here. What does `make install` do? There should be a `remove` rule somewhere if there is an `install` rule (though you have an `install` rule AND a `.install` file which seems weird.